packages/extension-redis/src/Redis.ts

```diff
@@ -241,12 +241,23 @@ export class Redis implements Extension {
     // to avoid conflict with other instances storing the same document.
     const resource = this.lockKey(documentName)
     const ttl = this.configuration.lockTimeout
-    const lock = await this.redlock.acquire([resource], ttl)
-    const oldLock = this.locks.get(resource)
-    if (oldLock) {
-      await oldLock.release
-    }
-    this.locks.set(resource, {lock})
+	try {
+    	await this.redlock.acquire([resource], ttl)
+    	const oldLock = this.locks.get(resource)
+      if (oldLock) {
+        await oldLock.release;
+      }
+	}  catch (error) {
+      //based on: https://github.com/sesamecare/redlock/blob/508e00dcd1e4d2bc6373ce455f4fe847e98a9aab/src/index.ts#L347-L349
+      if(error == 'ExecutionError: The operation was unable to achieve a quorum during its retry window.') {
+        // Expected behavior: Could not acquire lock, another instance locked it already.
+        // No further `onStoreDocument` hooks will be executed; should throw a silent error with no message.
+        throw new Error('', { cause: 'Could not acquire lock, another instance locked it already.' });
```

Comment:
due to [this](https://github.com/ueberdosis/hocuspocus/blob/de183c109521ac327ccf7254fd0155edb1d1ea27/packages/server/src/Hocuspocus.ts#L441-L443); I had to throw err with this struct 🤷